src/pages/Precaution/PrecautionTable/FormEditPrecaution.tsx

```diff
@@ -86,7 +80,7 @@ const FormEditPrecaution = ({
         setDistData(response.data);
       });
       setLoadingUserStatus(true);
-      await adminApi.getUsersForTable().then((response) => {
+      await precautionApi.getUsersForPrecaution().then((response) => {
```

Comment:
Do you really need await here, if it's your api call has await inside already

Reply:
Removed await statements in new commit.